tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -144,6 +145,10 @@ def visible_apps(self):
         return [self.InstalledApp(self.marionette, root_element)
                 for root_element in self.app_elements if root_element.is_displayed()]
 
+
+    def wait_for_homescreen_visible(self):
+        self.wait_for_element_displayed(*self._homescreen_container_locator)
```

Comment:
Making sure this is clear, this applies to the implementation call on line 150. The wrapper method standard is still wait_for_homescreen_visible().

Sorry for not catching this earlier myself.